doc/manual/src/installation/installing-binary.md

```diff
@@ -88,6 +88,28 @@ extension. The installer will also create `/etc/profile.d/nix.sh`.
 
 ### Linux
 
+To begin, if you are on Linux with systemd, remove the Nix daemon service:
+
+```console
+sudo systemctl stop nix-daemon.socket
+sudo systemctl stop nix-daemon.service
+sudo systemctl disable nix-daemon.socket
+sudo systemctl disable nix-daemon.service
+sudo systemctl daemon-reload
+```
+
+Then you can remove systemd service files:
+
+```console
+sudo rm -f /etc/systemd/system/nix-daemon.service /etc/systemd/system/nix-daemon.socket
```

Comment:
```suggestion
sudo rm /etc/systemd/system/nix-daemon.service /etc/systemd/system/nix-daemon.socket
```

Reply:
`-f` is to make sure no error is emitted if the file does not exist. But since this is intended to run line-by-line by a human, maybe it's fine without `-f`.

Reply:
Those files should exist if you installed the multi-user mode. Otherwise they might be somewhere else.